modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/release-notes/fragments/release-notes-tabs/index.html

```diff
@@ -6,12 +6,22 @@
 	[#assign currentTabId = (currentDisplayPaths?first)! /]
 
 	[#assign currentDisplayIds = (displayPortalURLs?last?split(currentTabId)?last)! /]
+
+	[#assign paginationParameterIndex = currentDisplayIds?index_of("&start=") /]
```

Comment:
@pedroliveirav I am wondering if we can use something like ${paramUtil.getString(request, "testParam", "defaultValue")} or another java util to handle and manipulate our url params easier. @kylembischof do you know of any helpful utils? I am just afraid that the regex and substring implementation is a little hard to read and fragile

Reply:
paramUtil sounds like the right helper class we usually use when working with url params. if we can use any helper besides regex that will be much more easily maintainable in the future

Reply:
@pedroliveirav is it possible to do Ryan's suggestion to switch to paramUtil instead of regex?

Reply:
I tried it, but I couldn't do it that way. If you have any more suggestions, I can try again. Thanks!

Reply:
The paramUtil.getString is working, but I can't set the value of the parameter in the manipulation, so I have to use the same resources I've already used, which doesn't seem feasible.